packages/purpose-process/test/suspendPurposeVersion.test.ts

```diff
@@ -571,4 +870,159 @@ describe("suspendPurposeVersion", () => {
       );
     }
   );
+  it("should throw organizationNotAllowed when the requester is the Consumer and is suspend a purpose version created by the delegate", async () => {
+    const mockEService = getMockEService();
+    const mockPurposeVersion: PurposeVersion = {
+      ...getMockPurposeVersion(),
+      state: purposeVersionState.active,
+    };
+    const mockPurpose: Purpose = {
+      ...getMockPurpose(),
+      eserviceId: mockEService.id,
+      versions: [mockPurposeVersion],
+    };
+    await addOnePurpose(mockPurpose);
+    await addOneEService(mockEService);
+
+    const delegation = getMockDelegation({
+      kind: delegationKind.delegatedConsumer,
+      eserviceId: mockEService.id,
+      delegateId: generateId<TenantId>(),
+      state: delegationState.active,
+    });
+
+    await addOneDelegation(delegation);
+
+    expect(
+      purposeService.suspendPurposeVersion(
+        {
+          purposeId: mockPurpose.id,
+          versionId: mockPurposeVersion.id,
+        },
+        {
+          authData: getRandomAuthData(delegation.delegateId),
```

Comment:
"the requester is the Consumer", so here there must be mockPurpose.consumerId
```suggestion
          authData: getRandomAuthData(mockPurpose.consumerId),
```